modules/gapi/samples/draw_example.cpp

```diff
@@ -0,0 +1,57 @@
+#include <opencv2/imgproc.hpp>
+#include <opencv2/videoio.hpp>
```

Comment:
are these needed?

Reply:
imgproc is required (put a comment), videoio is not (dropped).

BTW @TolyaTalamanov "render.hpp" is hardly usable without including "imgproc.hpp" manually -- why not include "imgproc.hpp" in "render.hpp" by default?

Reply:
@TolyaTalamanov can you please fix this after this thing is merged?

Reply:
Yes sure

Reply:
Great, thanks!